<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credentialWindow</class>
 <widget class="QDialog" name="credentialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Credential</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-credential-manager-Desktop-Release/icon.png</normaloff>../build-credential-manager-Desktop-Release/icon.png</iconset>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEntry">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>site-password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="siteEntry">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>site</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userEntry">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyEntry">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>encryption-key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPass">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>confirm password</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="messageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
